src/arrayMethodSort.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
     // write code here
+    for (let i = 0; i < this.length - 1; i++) {
```

Comment:
Create separate `sortString` that will be used if no `compareFunction` 

```suggestion
  [].__proto__.sort2 = function(compareFunction  = sortString) {
    // write code here
    for (let i = 0; i < this.length - 1; i++) {
```